test/runtests.jl

```diff
@@ -12,4 +12,9 @@ using Test
     @test generate_mesh(control_file) isa String
   end
 
+  @testset "generate_mesh()" begin
```

Comment:
```suggestion
  @testset "generate_mesh(; verbose=true)" begin
```

Reply:
Agreed, this was a copy paste error on my part